tracemap/tracemap-tool
# 88 beta adjustments

## Testing hints:
Follow these steps to test all functionalities:
1. delete your existing account from the user menu while logged in
  - you should not be able to login anymore.
2. try to create an account with an email not whitelisted
  - you should receive an error, that the email is not a beta user
3. create your account
  - you should receive an initial password per mail
4. login
5. change password and login again
  - you should automatically be logged out 2 seconds after seeing the success message
6. use the tool
  - you should be able to use the tool
7. logout while using the tool
  - you should be redirected to the /home route
8. try using the tool without being logged in (with a link to the tool)
  - you should receive an alert telling you that this is not possible and be redirected to /home route
9. login again
10. forget your password and request a new one
  - you should receive a mail with a link
11. follow the instructions until you have a new one
  - if clicking on the link in the mail you should see a text message that you received a new mail with your password
  - this mail is the same you receive when registering
12. reload the page or close the browser and revisit the page when logged in
  - you should stay logged in
13. login in two different browsers and use the tool in both
  - you should be able to do everything normally, the session_token is the same for both sessions for now. later we can alter this if we need to monitor multisessions but for now this is no problem.
